install/0000_90_cluster-version-operator_02_servicemonitor.yaml

```diff
@@ -16,3 +16,49 @@ spec:
   selector:
     matchLabels:
       k8s-app: cluster-version-operator
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    k8s-app: cluster-version-operator
+  name: cluster-version-operator
+  namespace: openshift-cluster-version
+spec:
+  groups:
+  - name: cluster-version
+    rules:
+    - alert: ClusterVersionOperatorDown
+      annotations:
+        message: ClusterVersionOperator has disappeared from Prometheus target discovery. Operator may be down or disabled, cluster will not be kept up to date and upgrades will not be possible.
```

Comment:
Would be tempted to say `Cluster version operator has ...` to make it flow as a message in more contexts.

Reply:
@smarterclayton done with https://github.com/openshift/cluster-version-operator/pull/232/commits/e7bc61e84d6d0f4f7161644378eed9478f58cc5d